Alexandrammr - Adding steps in instructions for #2150 (#6667)
github/docs: data/reusables/files/choose_pull_request.md

> 1. Once you create the Pull Request with your changes, in the "Pull Requests" list, click the pull request you'd like to merge.

Reviewer: ```suggestion
1. Create a pull request for your proposed changes.
```

Since the next action we want people to take is to create a pull request, I think we should be clear that they need to do that. So we can start this step by saying "create a pull request" and then the next step will be to merge the PR.

Author: @ethanpalm Thank you so much for all the suggestions, I've applied them. I did add a second step after the Create a PR, to say they need to navigate to the PR list to merge as it wasn't obvious to the users in the intial thread this issue resulted from. Let me know what you think, if you think it's not needed we can remove it 😁
